services/QuillLMS/app/services/gen_ai/activity_cloner.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module GenAI
+  class ActivityCloner < BaseCloner
+    ALPHA = Activity::ALPHA
+    BETA_GEN_AI = '[Beta - Gen AI]'
+    BRACKETS = /(\[.*\])/
+    CLONED_CLASS = ::Activity
+
+    COPYABLE_ATTRIBUTES = %w[
+      activity_classification_id
+      description
+      maximum_grade_level
+      minimum_grade_level
+      question_count
+      raw_score_id
+      repeatable
+      standard_id
+      topic_id
+    ].freeze
```

Comment:
Just wondering if we also want to copy relationships to `activity_category_activities` and `content_partner_activities` in this file (looks like other relevant associations are `belongs_to`, so those are accounted for in the `COPYABLE_ATTRIBUTES`.

Reply:
Great call on that.  Not sure how I missed those.

Reply:
Great call on those.  I only added in `belongs_to` relationships for the most part, but I think it's good to do another pass with the `has_many`